2018-05-16T02:37:29.358Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:29.359Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.310Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.311Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.383Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.384Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.469Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'giao@giaohangnhanh.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.470Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'giao@giaohangnhanh.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'giao@giaohangnhanh.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.470Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'thanh@glorious.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.470Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'thanh@glorious.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'thanh@glorious.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.510Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.511Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.654Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'long@hoanglong.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.655Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'long@hoanglong.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'long@hoanglong.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.885Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'hongan@vinhsang.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.885Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'hongan@vinhsang.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'hongan@vinhsang.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.957Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'chuyen@chuyenphatnhanh.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.957Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'chuyen@chuyenphatnhanh.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'chuyen@chuyenphatnhanh.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T02:37:30.986Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T02:37:30.986Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
